packages/components/table/stories/sortable_table.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import { Canvas, Story } from "@storybook/addon-docs";
+import LinkTo from '@storybook/addon-links/react';
+
+import { SortableTable } from "../src";
+
+<Meta
+  title="Components / SortableTable"
```

Comment:
I think we'll want this story within `Table`:
<img width="211" alt="Screenshot 2023-03-21 at 14 56 50" src="https://user-images.githubusercontent.com/54260/226699359-31696be5-814c-42ab-92eb-9528de09c651.png">

Reply:
I put this as a sibling following how we have ConnectionTable. My assumption for organization here is that ConnectionTable and SortableTable are there own components. @dgalarza is that how you see it?